neuralmonkey/evaluators/perplexity.py

```diff
@@ -16,3 +15,8 @@ def __call__(self, perplexities, _):
         """
 
         return np.mean(perplexities)
```

Comment:
To já právě nevim, ale vůbec to nesouvisí s timhle PR.. Vůbec jsem to nechtěl editovat, aby se náhodou nepřišlo na to, že je to tu zbytečný.. :-D

Nabízí se následující možnosti:
1. Můžeme celej perplexity evaluátor vyhodit.
2. ??
3. já myslim, že proto tam byl ten perplexity runner.. Asi de udělat runner, kterej udělá data series a to budou ty perplexity no.. Ale ten nemáme a ani nevim, proč by to někdo dělal, když to teď jde jednoduššejc

Takže jsem pro možnost 1